### PR TITLE
[14.0][MIG] purchase_request_analytic

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Migration to 14.0

This module adds the analytic account field to Purchase Requests.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review LGTM!

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

File: purchase_request_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline purchase_request_analytic/__manifest__.py:1]:
```suggestion
# Copyright 2019 ForgeFlow S.L.
```

File: purchase_request_analytic/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Request Analytic",
+    "version": "14.0.1.0.0",
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline purchase_request_analytic/__manifest__.py]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

File: purchase_request_analytic/__manifest__.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply purchase_request_analytic/__manifest__.py:1]: @MiquelRForgeFlow Done

File: purchase_request_analytic/__manifest__.py (hunk 2 above)
Comment (JuanyDForgeflow) [reply purchase_request_analytic/__manifest__.py]: @MiquelRForgeFlow Done

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**